src/components/Dashboard/CourtStats.js

```diff
@@ -1,20 +1,152 @@
 import React from 'react'
-import { Box, Distribution } from '@aragon/ui'
+import { Box, GU, textStyle, useTheme } from '@aragon/ui'
+import useCourtStats from '../../hooks/useCourtStats'
+import { formatUnits } from '../../lib/math-utils'
+import {
+  useANJBalanceToUsd,
+  useTokenBalanceToUsd,
+} from '../../hooks/useTokenBalanceToUsd'
```

Comment:
Let's put the hook imports together; right now they're separated by the `lib/math-utils` import